chart/skywalking/Chart.yaml

```diff
@@ -38,3 +38,7 @@ dependencies:
     version: 12.1.2
     repository: https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami
     condition: postgresql.enabled
+  - name: skywalking-banyandb-helm
+    version: 0.0.0-67df1c7
+    repository: oci://ghcr.io/apache/skywalking-banyandb-helm
```

Comment:
I don't think we could use this as the final release. We should request for docker hub repository, and do a release for banyandb helm. @hanahmily

Reply:
Because we deploy skywalking chart snapshot to ghcr so this makes sense in main branch but have to remember to change to a released one for banyandb chart before releasing.  This also implies that a skywalking chart release depends on a banyandb release.

Reply:
Makes sense to me. We will release the banyandb helm once the cluster mode is supported later. @ButterBright please make a note of this.

Reply:
> Because we deploy skywalking chart snapshot to ghcr so this makes sense in main branch but have to remember to change to a released one for banyandb chart before releasing. This also implies that a skywalking chart release depends on a banyandb release.

For me, the concern is, that we don't release `helm` repo in high frequency, so, most likely we definitely will lose track and release the next helm with this snapshot repository.

This PR is never urgent, so, if you feel the BanyanDB helm repo is ready to release, please go ahead to do so. I believe no matter banyandb helm repo is relying on BanyanDB server 0.5 release or not, it is close to be ready.

Reply:
> Makes sense to me. We will release the banyandb helm once the cluster mode is supported later. @ButterBright please make a note of this.

Okay.

Reply:
> > Because we deploy skywalking chart snapshot to ghcr so this makes sense in main branch but have to remember to change to a released one for banyandb chart before releasing. This also implies that a skywalking chart release depends on a banyandb release.
> 
> For me, the concern is, that we don't release `helm` repo in high frequency, so, most likely we definitely will lose track and release the next helm with this snapshot repository.
> 
> This PR is never urgent, so, if you feel the BanyanDB helm repo is ready to release, please go ahead to do so. I believe no matter banyandb helm repo is relying on BanyanDB server 0.5 release or not, it is close to be ready.

Let's wait for banyandb's first official release then.

Reply:
Agreed. @ButterBright, could you please release the banyandb helm chart version 0.1.0? @kezhenxu94, would you kindly guide them through the process?

Reply:
> Agreed. @ButterBright, could you please release the banyandb helm chart version 0.1.0? @kezhenxu94, would you kindly guide them through the process?

Reach out to me if there is any help I can offer, btw, it's too troublesome to ask a non-committer to release, even it's a committer who is releasing, one of the steps in release process still needs a PMC member to help, so basically there is few stuff can be done by @ButterBright to propose a release yet.

Reply:
We can't accept non committer to do a release. Because ASF requires apache ID and his/her sign based on that.
@hanahmily You need to take the responsibility to be release manager.

Reply:
I will take care of the release